src/stories/colors/colorNamesMap.js

```diff
@@ -0,0 +1,103 @@
+const colorNamesMap = {
```

Comment:
This part feels like overkill.
Longer term, I'd like to configure postcss-custom-properties to create js files from our shared css variable files.
That will require a bit of digging, but I mostly didn't do it now because I don't want to create confusion with the existing `shared/colors.js` until @mdespuits and I have confirmed it's not being used anywhere.